### DISCUSSION TITLE
change in appearance

### DISCUSSION THREAD

**Discussion by ABOD2005m** [libre-tube/LibreTube#3642] [General]
Please change the bottom bar

**Comment by Silther** [comment]
Can you elaborate on what you mean?

**Comment by akano12** [comment]
For now, it's better to spend the limited time devs have on addressing bugs.

Pull Requests that address UI changes are always welcomed. In case you can submit a concept/design using figma or any other similar service, I'm pretty sure someone can convert it to a Kotlin code
